generator/04-syndesis-keycloak.yml.mustache

```diff
@@ -93,35 +93,10 @@
           app: syndesis
           component: syndesis-keycloak
           deploymentconfig: syndesis-keycloak
-        annotations:
```

Comment:
This does not work on minishift. It seems that it still needs the annotation based init-container specfication. When I apply the new templates without annotation, I get the following DC:

```yml
apiVersion: v1
kind: DeploymentConfig
metadata:
  annotations:
    openshift.io/generated-by: OpenShiftNewApp
  creationTimestamp: 2017-10-01T07:59:26Z
  generation: 2
  labels:
    app: syndesis
    component: syndesis-keycloak
  name: syndesis-keycloak
  namespace: myproject
  resourceVersion: "1616"
  selfLink: /oapi/v1/namespaces/myproject/deploymentconfigs/syndesis-keycloak
  uid: 721ec1ef-a67e-11e7-8b4c-1e88dd619605
spec:
  replicas: 1
  selector:
    app: syndesis
    component: syndesis-keycloak
    deploymentconfig: syndesis-keycloak
  strategy:
    activeDeadlineSeconds: 21600
    recreateParams:
      timeoutSeconds: 600
    resources:
      limits:
        memory: 256Mi
      requests:
        memory: 20Mi
    type: Recreate
  template:
    metadata:
      annotations:
        openshift.io/generated-by: OpenShiftNewApp
      creationTimestamp: null
      labels:
        app: syndesis
        component: syndesis-keycloak
        deploymentconfig: syndesis-keycloak
    spec:
      containers:
      - args:
        - -b
        - 0.0.0.0
        - -Dkeycloak.import=/opt/jboss/keycloak/standalone/configuration/syndesis-realm.json
        - -Dkeycloak.migration.strategy=IGNORE_EXISTING
        - -Djavax.net.ssl.trustStore=/opt/jboss/keycloak/standalone/configuration/tls/openshift-truststore.jks
        env:
        - name: KEYCLOAK_USER
          valueFrom:
            secretKeyRef:
              key: username
              name: syndesis-keycloak-admin
        - name: KEYCLOAK_PASSWORD
          valueFrom:
            secretKeyRef:
              key: password
              name: syndesis-keycloak-admin
        image: docker.io/syndesis/keycloak-openshift:2.5.4.Final
        imagePullPolicy: IfNotPresent
        livenessProbe:
          failureThreshold: 3
          httpGet:
            path: /auth/
            port: 8080
            scheme: HTTP
          initialDelaySeconds: 180
          periodSeconds: 10
          successThreshold: 1
          timeoutSeconds: 1
        name: syndesis-keycloak
        ports:
        - containerPort: 8080
          protocol: TCP
        readinessProbe:
          failureThreshold: 3
          httpGet:
            path: /auth/
            port: 8080
            scheme: HTTP
          initialDelaySeconds: 10
          periodSeconds: 10
          successThreshold: 1
          timeoutSeconds: 1
        resources:
          limits:
            memory: 768Mi
          requests:
            memory: 480Mi
        terminationMessagePath: /dev/termination-log
        terminationMessagePolicy: File
        volumeMounts:
        - mountPath: /opt/jboss/keycloak/standalone/configuration/standalone.xml
          name: syndesis-keycloak-config
          subPath: config/standalone.xml
        - mountPath: /opt/jboss/keycloak/standalone/configuration/syndesis-realm.json
          name: syndesis-keycloak-config
          subPath: config/syndesis-realm.json
        - mountPath: /opt/jboss/keycloak/standalone/configuration/tls
          name: syndesis-keycloak-tls
      dnsPolicy: ClusterFirst
      restartPolicy: Always
      schedulerName: default-scheduler
      securityContext: {}
      terminationGracePeriodSeconds: 30
      volumes:
      - configMap:
          defaultMode: 420
          items:
          - key: standalone.xml
            path: config/standalone.xml
          - key: syndesis-realm.json
            path: config/syndesis-realm.json
          name: syndesis-keycloak-config
        name: syndesis-keycloak-config
      - emptyDir: {}
        name: syndesis-keycloak-tls
  test: false
  triggers:
  - type: ConfigChange
status:
  availableReplicas: 0
  conditions:
  - lastTransitionTime: 2017-10-01T07:59:26Z
    lastUpdateTime: 2017-10-01T07:59:26Z
    message: Deployment config does not have minimum availability.
    status: "False"
    type: Available
  - lastTransitionTime: 2017-10-01T07:59:27Z
    lastUpdateTime: 2017-10-01T07:59:27Z
    message: replication controller "syndesis-keycloak-1" is waiting for pod "syndesis-keycloak-1-deploy"
      to run
    status: Unknown
    type: Progressing
  details:
    causes:
    - type: ConfigChange
    message: config change
  latestVersion: 1
  observedGeneration: 2
  replicas: 1
  unavailableReplicas: 1
  updatedReplicas: 1
```

Reply:
This might be also the reason why the CI is broken ?

Reply:
Confused by this. Works perfectly for me on local openshift cluster with openshift 3.6.0, which i thought was our target openshift version?